src/index.js

```diff
@@ -1,60 +1,68 @@
+/* eslint-disable max-classes-per-file */
+
 import './style.css';
+import Store from '../modules/Store';
+import Ui from '../modules/Ui';
+import List from '../modules/List';
 import '@fortawesome/fontawesome-free/js/fontawesome';
 import '@fortawesome/fontawesome-free/js/solid';
 import '@fortawesome/fontawesome-free/js/regular';
 import '@fortawesome/fontawesome-free/js/brands';
 
+// variables
+
 const toDoList = document.querySelector('.to-do-list');
-const listItems = [
-  {
-    description: 'Feed the dog',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Make breakfast',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Get grocery',
-    completed: false,
-    index: 2,
-  },
-];
-
-for (let i = 0; i < listItems.length; i += 1) {
-  const newItem = document.createElement('li');
-  newItem.setAttribute('id', i);
-  newItem.innerHTML = `
-    <div>
-    <input type="checkbox" class="items-list"> 
-    <label> ${listItems[i].description}</label><br>
-    </div>
-    <i class="fa-solid fa-ellipsis-vertical"></i>
-    `;
-  toDoList.appendChild(newItem);
-}
-
-// listItems.forEach((item) => {
-//   count++;
-//   const newItem = document.createElement('li');
-//   newItem.setAttribute('id', count - 1)
-//   newItem.innerHTML = `
-//     <div>
-//     <input type="checkbox" class="items-list">
-//     <label> ${item.description}</label><br>
-//     </div>
-//     <i class="fa-solid fa-ellipsis-vertical"></i>
-//     `;
-
-//   toDoList.appendChild(newItem);
-// });
-
-const clearCompletedBtn = document.createElement('li');
-clearCompletedBtn.classList.add('clear-completed');
-clearCompletedBtn.innerHTML = `
-  <span>Clear all completed</span>
-`;
-
-toDoList.appendChild(clearCompletedBtn);
+
+document.addEventListener('DOMContentLoaded', Ui.DisplayList());
+
+const addInput = document.querySelector('#add-list');
+addInput.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter') {
+    const Lists = Store.getLists();
+    const inputValue = document.querySelector('#add-list').value;
+    const list = new List(inputValue, Lists.length + 1);
+    Store.setLists(list);
+    Ui.addLists(list);
+  } else {
+    return false;
+  }
+  addInput.value = '';
+  return true;
```

Comment:
- [ ] [Optional] I can see that you are repeating these lines of code twice, it would be nice to create a function and execute these lines of code in it and call the function whenever you need to. Doing it this way will ensure that you are applying the [DRY](https://codinglead.co/javascript/what-is-DRY-code) principle in coding. 👍